recipe/meta.yaml

```diff
@@ -8,21 +8,22 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastcore-{{ version }}.tar.gz
-  sha256: aa8cdb3064ced1fd7430a0cb15430cd2f99a4c9e932330c79614229913905697
+  sha256: f1a2eb04eb7933f3f9eb4064852817df44dc96e20fab5658c14c035815269a3f
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
+    - wheel
   run:
     - packaging
```

Comment:
```suggestion
```
I don't see fastcore importing packaging. Same as pip, this should be removed.

Reply:
> I don't see fastcore importing packaging. Same as pip, this should be removed.

`pip check` fails without packaging. It is specified as a dependency in the upstream, is it preferred to override that? Also, we are installing with pip, should pip be a host dependency??

Reply:
`pip` and `packaging` are runtime dependencies https://github.com/fastai/fastcore/blob/889d407805e1931d480b83174a057179abbdc7ea/setup.py#L29

Reply:
This looks to me to be an error from upstream. I think we should patch setup.py to remove runtime dependencies on pip and packaging. (and set pip and packaging in host).

Reply:
I agree they seem to be not run dependencies and there are no mentions in the source code